rs/p2p/quic_transport/src/connection_manager.rs

```diff
@@ -261,6 +258,25 @@ impl ConnectionManager {
         self.node_id < *dst
     }
 
+    // Conditions under which the node can start outbound connecting attempt
+    // - the node is a designated dialer
+    // - peer is in the subnet
+    // - this node is not part of the subnet (can happen when a node is removed from the subnet)
```

Comment:
The node should be in the subnet. This is also the condition you check in the code.

Reply:
It is checked, right ?

Reply:
But your comment bullet point says the opposite?